sgkit/io/vcf/vcf_reader.py

```diff
@@ -331,6 +332,10 @@ def vcf_to_zarr_sequential(
     region: Optional[str] = None,
     chunk_length: int = 10_000,
     chunk_width: int = 1_000,
+    compressor: Optional[Any] = Blosc(
```

Comment:
Better to have a DEFAULT_COMPRESSOR module variable, so that we can at least deal with missing compressors? Maybe it should be None in the signature also, so that we can have the freedom to choose different compressors for different fields, if we want to? (We probably will, I would imagine)

Reply:
Changed to have a `DEFAULT_COMPRESSOR` variable, but I haven't made None the default, since it is used to signify no compression. Having different compressors for different fields can be achieved by setting `encoding` - users can do this now, and the library could change the defaults in future if we wanted.

Reply:
Sounds good. I guess we can check if the default was specified by saying ``compressor is DEFAULT_COMPRESSOR``, right, since this is a singleton?